README.md

```diff
@@ -75,3 +75,13 @@ const CHAINLINK_HTTP_GET_JSON_PARSE_JOB_ID = "76ca51361e4e444f8a9b18ae350a5725";
 const PAYMENT = 1;
 ```
 https://ropsten.explorer.chain.link/
+
+### Deployment
+register ethereum name on *ens link here*
+requires ipfs
+
+* run yarn build inside of web/
+* ipfs add -f build/
+* take last hash displayed and add it to content
+
+upload may take some time
```

Comment:
upload may take some time to propagate and be available.

Reply:
add the following:

Followed directions [here](https://medium.com/the-ethereum-name-service/how-to-host-your-dapp-with-ipfs-ens-and-access-it-via-ethdns-c96046059d87)